diff --git a/dist/chunk-7UHTFYFF.mjs b/dist/chunk-7UHTFYFF.mjs
index 67a4db976aa0e79a069dcbc083a1fa029275bf75..509fe95fb42c615d91c011bcb5e47035a1d802c4 100644
--- a/dist/chunk-7UHTFYFF.mjs
+++ b/dist/chunk-7UHTFYFF.mjs
@@ -323,18 +323,22 @@ function prettify(data) {
   data.slides.forEach(prettifySlide);
   return data;
 }
+function safeParseYAML(str) {
+  const res = YAML.load(str)
+  return isObject(res) ? res : {}
+}
 function matter(code) {
-  let data = {};
-  const content = code.replace(
-    /^---.*\r?\n([\s\S]*?)---/,
-    (_, d) => {
-      data = YAML.load(d);
-      if (!isObject(data))
-        data = {};
-      return "";
-    }
-  );
-  return { data, content };
+  const data = {}
+  const replacer = (_, d) => {
+    Object.assign(data, safeParseYAML(d))
+    return ''
+  }
+
+  const content = code
+    .replace(/^---.*\r?\n([\s\S]*?)---/, replacer)
+    .replace(/^\s*```ya?ml([\s\S]*?)```/, replacer)
+
+  return { data, content }
 }
 function detectFeatures(code) {
   return {
